fili-wikipedia-example/src/main/java/com/yahoo/wiki/webservice/data/config/metric/WikiMetricLoader.java

```diff
@@ -66,24 +68,25 @@ public void loadMetricDictionary(MetricDictionary metricDictionary) {
         buildMetricMakers(metricDictionary);
 
         // Metrics that directly aggregate druid fields
-        List<MetricInstance> metrics;
-        metrics = Arrays.asList(
-                new MetricInstance(WikiApiMetricName.COUNT, countMaker),
-                new MetricInstance(WikiApiMetricName.ADDED, doubleSumMaker, WikiDruidMetricName.ADDED),
-                new MetricInstance(WikiApiMetricName.DELETED, doubleSumMaker, WikiDruidMetricName.DELETED),
-                new MetricInstance(WikiApiMetricName.DELTA, doubleSumMaker, WikiDruidMetricName.DELTA)
+        List<MetricInstance> metrics = Arrays.asList(
+                new MetricInstance(new LogicalMetricInfo(WikiApiMetricName.COUNT.asName()), countMaker),
+                new MetricInstance(
+                        new LogicalMetricInfo(WikiApiMetricName.ADDED.asName()),
+                        doubleSumMaker,
+                        WikiDruidMetricName.ADDED
+                ),
+                new MetricInstance(
+                        new LogicalMetricInfo(WikiApiMetricName.DELETED.asName()),
+                        doubleSumMaker,
+                        WikiDruidMetricName.DELETED),
+                new MetricInstance(
+                        new LogicalMetricInfo(WikiApiMetricName.DELTA.asName()),
+                        doubleSumMaker,
+                        WikiDruidMetricName.DELTA
+                )
```

Comment:
The constructor `new MetricInstance(apiMetricName, doubleSumMaker, fieldName)` has been deprecated in 2017. New constructor that takes a `LogicalMetricInfo` is used here.